string target_name
geometry_msgs/Point distance
---
bool result
